docs/samples.rst

```diff
@@ -3684,4 +3688,39 @@ Request a sales report from a certain date to another date, issued in all office
 
     $salesReportResult = $client->salesReportsDisplayQueryReport($opt);
 
+-------------------------
```

Comment:
AFAIK in RST your lines above and below titles need to be of the exact same length of the text.

See http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections